domains/auth/Auth.API/Tests/Services/OidcProviders/SignaturGruppenIT.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Net;
+using System.Net.Http;
+using API.Configuration;
+using API.Services;
+using API.Services.OidcProviders;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Moq;
+using RichardSzalay.MockHttp;
+using Xunit;
+using Xunit.Categories;
+
+namespace Tests.Services.OidcProviders;
+
+[UnitTest]
+public class SignaturGruppenIT
+{
+    private readonly Mock<IOidcService> _mockOidcService = new();
+    private readonly Mock<ILogger<SignaturGruppen>> _mockLogger = new();
+    private readonly MockHttpMessageHandler _handlerMock = new();
+    private readonly Mock<IOptions<AuthOptions>> _mockAuthOptions = new();
+
+    private SignaturGruppen _signaturGruppen;
+
+    public SignaturGruppenIT()
+    {
+        _mockAuthOptions.Setup(a => a.Value).Returns(new AuthOptions()
+        {
+            OidcUrl = "http://localhost:8080"
+        });
+
+        _signaturGruppen = new SignaturGruppen(
+            _mockLogger.Object,
+            _mockOidcService.Object,
+            _mockAuthOptions.Object,
+            new HttpClient(_handlerMock)
+        );
+
+    }
+
+    [Fact]
+    public async void can_logout_from_signaturgruppen()
+    {
+        _handlerMock.When("/api/v1/session/logout").Respond(new HttpResponseMessage(HttpStatusCode.OK));
+        await _signaturGruppen.Logout("test");
+
+        _mockLogger.Verify(logger => logger.Log(
```

Comment:
Currently it is only verified that a logger message is written or not written in these tests. But I think that it is more important that

* the result from _signaturGruppen.Logout("test") is an OkResult in both cases
* /api/v1/session/logout on SignaturGruppen is called in both cases

Reply:
fixed